benchmark/daisy/daisy.jl

```diff
@@ -48,23 +48,24 @@ for name in keys(DAISY_ND)
     RELPREC[name] = Dict()
     dom = DAISY_ND[name]["dom"]
     ref = DAISY_ND[name]["ref"]
-    for m in [2, 5, 10]
+    for ord in [2, 5, 10]
         varnames, numvars = DAISY_ND[name]["vars"], DAISY_ND[name]["numvars"]
-        vars = set_variables(Float64, varnames, order=m, numvars=numvars)
+
+        vars = set_variables(Float64, varnames, order=ord, numvars=numvars)
```

Comment:
This should be changed to `vars = set_variables(Float64, varnames, order=2*ord, numvars=numvars)`.   
The multi-variate case needs `2*ord` in order to get a *tigher* bound of the product in a performant way; this is done internally for the univariate case, which is why this is not needed there.

Reply:
I have tried it locally, will add it in the next commit.